shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -119,6 +132,17 @@ static void OnPlatformMessage(const FlutterPlatformMessage* message, FlutterEngi
   [engine engineCallbackOnPlatformMessage:message];
 }
 
+static bool OnAcquireExternalTexture(FlutterEngine* engine,
+                                     int64_t texture_identifier,
+                                     size_t width,
+                                     size_t height,
+                                     FlutterOpenGLTexture* openGLTexture) {
```

Comment:
Nit: The style guide allows using either C++-style variable naming or ObjC-style variable naming in a function, but it needs to be consistent at least within the function (and ideally file, but I don't think there's a convention yet in this file so either is fine).

Reply:
This wasn't addressed; fixing now.